keyboards/crkbd/rev2/config.h

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2023 @foostan
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#define SERIAL_USART_FULL_DUPLEX
+#define SERIAL_USART_TX_PIN GP12
+#define SERIAL_USART_RX_PIN GP13
+#define SPLIT_HAND_PIN GP21
+
+#define SPLIT_TRANSPORT_MIRROR
+#define SPLIT_LAYER_STATE_ENABLE
+#define SPLIT_MODS_ENABLE
+
+#define NOP_FUDGE 1
+
+#define I2C_DRIVER I2CD1
+#define OLED_UPDATE_INTERVAL 50
+
+#define WS2812_DI_PIN GP10
```

Comment:
```suggestion
#define I2C_DRIVER I2CD1
#define OLED_UPDATE_INTERVAL 50
```

Reply:
Where are deleted parameters set? It does not work well if I just delete them at least.

Reply:
Ah, this one https://github.com/qmk/qmk_firmware/pull/22676#discussion_r1428670937. Thanks!